test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -0,0 +1,451 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using NuGet.XPlat.FuncTest;
+using NuGet.Test.Utility;
+using NuGet.Packaging;
+using NuGet.Packaging.Core;
+using NuGet.Packaging.PackageExtraction;
+using NuGet.Protocol;
+using Xunit;
+
+namespace Msbuild.Integration.Test
+{
+    public class MsbuildIntegrationTestFixture : IDisposable
+    {
+        private readonly TestDirectory _cliDirectory;
```

Comment:
Is this necessary? 

You don't need to copy the CLI, it'll just slow down your tests :)